src/lib/storage/chunk_encoder.hpp

```diff
@@ -29,6 +29,13 @@ struct SegmentEncodingSpec {
 
 using ChunkEncodingSpec = std::vector<SegmentEncodingSpec>;
 
+/**
+ * Create a ChunkEncodingSpec. For each Column in @param table, check whether the @param desired_segment_encoding
+ * supported. If it is supported, use it for this column. If it is not, use EncodingType::Unencoded for this column.
+ */
+ChunkEncodingSpec create_compatible_chunk_encoding_spec(const Table& table,
```

Comment:
Mmh, this looks like a test helper method that might be too specific and nondeterministic to be called in the actual code. Should we move it into src/test?

Reply:
I'd rather have code in lib/benchmark lib, but this is minor, so I will move it to BaseTest.